test-cases/tests/performance/k04-run-performance-test-against-rhoam.md

```diff
@@ -0,0 +1,101 @@
+
+- create a `perf-test-start-time.txt` file as described in [capture_resource_metrics script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/capture_resource_metrics.sh).
+  - the actual performance test run doesn't start immediately, the performance test suite creates various 3scale (Product, Backend, Application, Application Plan) and SSO (realm, client, users) entities first
+  - best to track the log of Hyperfoil controller to get the exact time the when the `rampUp` phase starts
+  - create the file in the directory where the script resides
+
+7. Create a `perf-test-end-time.txt` for [capture_resource_metrics script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/capture_resource_metrics.sh)
+
+- create the file in the directory where the script resides
+- to get the exact time track the Hyperfoil controller log
+
+8. Collect the data about the performance test run
+
+- from [alerts-during-perf-testing](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/alerts-during-perf-testing.sh)
+- from Hyperfoil
+  - install Hyperfoil locally
+  - bin/cli.sh
+  - connect <hyperfoil-url-without-protocol> -p <port-8090-is-default>
+  - runs # to see all the runs
+  - status <your-run-name>
+  - stats <your-run-name>
+  - export -f json -d . <your-run-name> # to export the data
+  - use [report tool](https://github.com/Hyperfoil/report) to generate the HTML out of the exported data
+- eye review of various Grafana Dashboards, see [this guide](https://docs.google.com/document/d/1KznoB-we73lGUViJApVHyBoIgh3xpgyak6ODAEAHbwk/edit?usp=sharing) on how to do it
+- use [capture_resource_metrics script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/capture_resource_metrics.sh) to get the data
+- add a new column about the run to the [Load Testing](https://docs.google.com/spreadsheets/d/1v_bZIk8B_thZi93hGBNiOOnbSix0gmpwy3LV_s4WFPw/edit?usp=sharing) spreadsheet
+  - fill in the first few rows with all the relevant information about the benchmark used
+  - rest of the rows should be filled in based on the [capture_resource_metrics script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/capture_resource_metrics.sh)
+  - add any additional info (e.g. links to Hyperfoil report, Grafana Dashboard snapshots etc)
+
+9. Analyse the results
+
+- compare with the previous runs
+
+10. Attach the spreadsheet to the JIRA ticket
+
+- store the spreadsheet in [Google Drive](https://drive.google.com/drive/folders/1DR2qV7Z2bbvJv47Vnah-n_Qm5lnlcQfd)
```

Comment:
It's going to be just one spreadsheet, so probably no need for this step

Reply:
Right, removed.